include/fluent-bit/flb_event.h

```diff
@@ -29,7 +29,10 @@
 #define FLB_EVENT_TYPE_METRICS    FLB_INPUT_CHUNK_TYPE_METRICS
 #define FLB_EVENT_TYPE_TRACES     FLB_INPUT_CHUNK_TYPE_TRACES
 #define FLB_EVENT_TYPE_PROFILES   FLB_INPUT_CHUNK_TYPE_PROFILES
+
+#ifdef FLB_HAVE_SQLDB
```

Comment:
Please remove this conditional, this should be handled at a different level.

Reply:
I will restore it, can you answer on https://github.com/fluent/fluent-bit/pull/10239#issuecomment-2906988713, not clear where I should add a warning to indicate that this is not supported.

Reply:
I've restored it.